data/mapcss-mapping.csv

```diff
@@ -1671,3 +1671,5 @@ amenity|parking|underground|private;[amenity=parking][location=underground][acce
 amenity|parking|street_side|private;[amenity=parking][parking=street_side][access=private];;name;int_name;1590;
 amenity|parking|lane|private;[amenity=parking][parking=lane][access=private];;name;int_name;1591;
 amenity|compressed_air;1592;
+fee|yes;[fee?];;name;int_name;1593;
+fee|no;[fee=no];;name;int_name;1594;
```

Comment:
Let's move them up, somewhere near 500 instead of current deprecated?
I'd prefer to put _very_ common used types with small values.

Reply:
Moved to 500 and 501

Reply:
A note for the future: regeneration is needed after `mapcss-mapping.csv` changes (atm the types order in `data/types.txt` is wrong on master).